css/main.css

```diff
@@ -619,48 +630,63 @@ main.content {
   }
 }
 
+#book-selection-menu-book-list {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.select-chapter #book-selection-menu-book-list {
+  display: none;
+}
+
 .book-list-column {
-  float: left;
-  width: 10.25em;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(10, 1fr);
+  grid-auto-flow: column;
   list-style-type: none;
-  padding: 0.8em;
-  padding-bottom: 0;
+  padding: 0.3em 0.8em;
   margin-top: 0;
   margin-bottom: 0;
+  text-align: center;
 }
 
-.ot-column {
-  border-bottom: 1px solid lightgrey;
-  height: 26.5em;
-}
-
-.book-list-column li {
-  margin-bottom: 0.7em;
+.nt-books {
+  grid-template-rows: repeat(9, 1fr);
 }
 
 .ot-nt-divider {
-  clear: both;
+  margin-top: 0.2em;
+  border-top: 1px solid lightgray;
 }
 
+.book-list-column li{
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  line-height: 2em;
+}
 
 /* Formatting for large tablets and desktop */
-
 @media only screen and (min-width: 770px) {
-  .ot-column {
-    border-bottom: unset;
-    height: unset;
+  #book-selection-menu-book-list {
+    flex-direction: row;
   }
-  .book-list-column {
-    width: unset;
+
+  .nt-books {
+    grid-template-columns: repeat(3, 1fr);
+    flex-basis: 45%; /* 3 columns of 7 column layout + divider */ 
```

Comment:
adjust this to make the columns equal width

Reply:
Thanks!